src/main/java/core/basesyntax/service/impl/CsvFileTransactionReaderService.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionReaderService;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileTransactionReaderService implements TransactionReaderService {
+    private final String fileName;
+
+    public CsvFileTransactionReaderService(String fileName) {
+        this.fileName = fileName;
+    }
```

Comment:
It's not good to create service that can read only 1 predefined file.
Please make filename method argument, not constructor